test/erc20_gateway/erc20_cogateway/withdraw.js

```diff
@@ -0,0 +1,163 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const BN = require('bn.js');
+
+const ERC20Cogateway = artifacts.require('ERC20CogatewayDouble');
+
+const { AccountProvider } = require('../../test_lib/utils.js');
+const MessageBusUtils = require('../../message_bus/messagebus_utils');
+const Utils = require('../../test_lib/utils.js');
+
+contract('ERC20Cogateway::withdraw', (accounts) => {
+  const accountProvider = new AccountProvider(accounts);
+  let erc20Cogateway;
+  let utilityToken;
+  let setupGenesisParams = {};
+  let withdrawParam = {};
+  let withdrawIntent;
+  let sender;
+  let outboundChannelIdentifier;
+
+  const WITHDRAW_INTENT_TYPEHASH = web3.utils.keccak256(
```

Comment:
`'web3' is not defined.`
How is it working?

Reply:
Fixed. It was working maybe because of web3 available in truffle.